packit/local_project.py

```diff
@@ -89,7 +90,7 @@ def __init__(
         """
         self.working_dir_temporary = False
         self.git_repo: git.Repo = git_repo
-        self.working_dir: str = working_dir
+        self.working_dir: Path = Path(working_dir)
```

Comment:
As we discussed with @csomh, it would be good to have `Path` also in the argument. What do you think? (It may cause another wave of rewrites.)